core/java/com/google/instrumentation/stats/RpcConstants.java

```diff
@@ -116,29 +113,27 @@
           MeasurementUnit.create(6, bytes));
 
   // Common histogram bucket boundaries for bytes received/sets DistributionViewDescriptors.
-  static final List<Double> RPC_BYTES_BUCKET_BOUNDARIES = Arrays.asList(
-        0.0, 1024.0, 2048.0, 4096.0, 16384.0, 65536.0, 262144.0, 1048576.0, 4194304.0, 16777216.0,
-        67108864.0, 268435456.0, 1073741824.0, 4294967296.0);
+  static final List<Double> RPC_BYTES_BUCKET_BOUNDARIES = Collections.unmodifiableList(
+      Arrays.asList(0.0, 1024.0, 2048.0, 4096.0, 16384.0, 65536.0, 262144.0, 1048576.0, 4194304.0,
+          16777216.0, 67108864.0, 268435456.0, 1073741824.0, 4294967296.0));
 
   // Common histogram bucket boundaries for latency and elapsed-time DistributionViewDescriptors.
-  static final List<Double> RPC_MILLIS_BUCKET_BOUNDARIES = Arrays.asList(
-        0.0, 1.0, 2.0, 3.0, 4.0, 5.0, 6.0, 8.0, 10.0, 13.0, 16.0, 20.0, 25.0, 30.0, 40.0, 50.0,
-        65.0, 80.0, 100.0, 130.0, 160.0, 200.0, 250.0, 300.0, 400.0, 500.0, 650.0, 800.0, 1000.0,
-        2000.0, 5000.0, 10000.0, 20000.0, 50000.0, 100000.0);
-
-  /**
-   * Census defined rpc client {@link ViewDescriptor}s.
-   */
+  static final List<Double> RPC_MILLIS_BUCKET_BOUNDARIES = Collections.unmodifiableList(
+      Arrays.asList(0.0, 1.0, 2.0, 3.0, 4.0, 5.0, 6.0, 8.0, 10.0, 13.0, 16.0, 20.0, 25.0, 30.0,
+          40.0, 50.0, 65.0, 80.0, 100.0, 130.0, 160.0, 200.0, 250.0, 300.0, 400.0, 500.0, 650.0,
+          800.0, 1000.0, 2000.0, 5000.0, 10000.0, 20000.0, 50000.0, 100000.0));
+
+  // Census defined rpc client {@link ViewDescriptor}s.
   public static final DistributionViewDescriptor RPC_CLIENT_ERROR_COUNT_VIEW =
       DistributionViewDescriptor.create(
-          "rpc client error count",
+          "rpc client error_count",
```

Comment:
Why have underscores just for the last bit of the name? I'd expect all underscores, or none.